src/styles/sidebar/components/focused-mode-header.scss

```diff
@@ -1,11 +1,26 @@
+// A dark grey button used for the primary action
+// in a form
 .focused-mode-header {
-  margin-bottom: 10px;
-  button {
-    width: 100%;
-    height: 24px;
-    margin-right: 10px;
-    &:focus {
-      outline: none;
+  display: flex;
+  align-items: center;
+
+  &__btn {
+    @include primary-action-btn;
+    @include outline-on-keyboard-focus;
+    display: flex;
+    padding-left: 6px;
```

Comment:
The paddings here are fiddly and specific; if there's a more harmonious way to do this I'm open.